api/unittest/ocapitest.cpp

```diff
@@ -106,16 +100,7 @@ class ApiHelper {
     oc_add_resource(s_pResource);
   }
 
-  static void signalEventLoop(void)
-  {
-#ifdef _WIN32
-    WakeConditionVariable(&s_cv);
-#else
-    pthread_mutex_lock(&s_mutex);
-    pthread_cond_signal(&s_cv);
-    pthread_mutex_unlock(&s_mutex);
-#endif
-  }
```

Comment:
is it correct ?
```
#ifdef _WIN32
    WakeConditionVariable(&s_cv);
#else
    pthread_cond_signal(&s_cv);
#endif
```

Reply:
in this file, neither the conditional variable nor the mutex were used anywhere else. probably a copy and paste from some other test.

Reply:
Ok pls check if the code is used at all. If not remove it.

Reply:
Other callbacks must be set and this one cannot be NULL, so there must be an empty implementation like this.